tikv/pd
# pd-ctl: make use of the new bundle api

### What problem does this PR solve?

Close #2863. Add a set of commands to pd-ctl, which make use of the new bundle api.

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)

1. `placement-rule rule-bundle load` to load the default rule.
2. modify the rule bundle, and `placement-rule rule-bundle save`, then use `load` check
3. create a new bundle, using `rule-bundle set`, then use `load` check
4. use `rule-bundle get` to check all rules
5. use `rule-bundle delete` to delete one group, and use `load` check

### Release note

- pd-ctl supports altering placement rules in bundles/groups
